security_check/main.py

```diff
@@ -44,31 +43,27 @@ def _check_image(image, severity, whitelist):
                          vuln.get('vulnerability'))
             continue
         if _filter_severity(vuln['severity'], severity):
-            unpatched += 1
+            unpatched[vuln['vulnerability']] = vuln
 
     if unpatched:
-        base_unpatched = 0
+        base_unpatched = {}
         img = parsed.get('image_analysis')
         if img:
             base_img_url = img[0]['base_image_url']
             base_image = base_img_url[len('https://'):base_img_url.find('@')]
             base_unpatched = _check_image(base_image, severity, whitelist)
-        unpatched -= base_unpatched
-        logging.info('Found %s unpatched vulnerabilities in %s. Run '
-                     '[gcloud beta container images describe %s] '
-                     'to see the full list.',
-                     unpatched, image, full_name)
+        for vuln in base_unpatched.keys():
+            del unpatched[vuln]
```

Comment:
Done. I'm gonna forego the flag since we're the only users of this script currently.